srcpkgs/wyrd/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'wyrd'
+pkgname=wyrd
+version=1.5.3
+revision=1
+build_style=gnu-configure
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+hostmakedepends="ocaml camlp5 ocaml-curses texlive texlive-LuaTeX python opam base-devel"
```

Comment:
Can this be built with python3?  We're trying to avoid python2.x going forward.

Also, I believe `base-devel` isn't needed here.

Reply:
Good catch, I'll try right switching it to python3 now. I think some of the development tools starting popping up one after another and so I just used `base-devel` as a catchall. I'll be more explicit instead.